server/lib/src/filter.rs

```diff
@@ -1643,6 +1666,7 @@ impl FilterResolved {
             | FilterResolved::And(_, sf)
             | FilterResolved::Inclusion(_, sf)
             | FilterResolved::AndNot(_, sf) => *sf,
+            FilterResolved::Invalid(_) => Some(NonZeroU8::new(1).unwrap()),
```

Comment:
Comment that because this always returns false there is no index cost so it's always 1.